Source/SLKTextViewController.m

```diff
@@ -1397,7 +1400,8 @@ - (void)slk_willShowOrHideKeyboard:(NSNotification *)notification
     
     // Begin and end frames are the same when the keyboard is shown during navigation controller's push animation.
     // The animation happens in window coordinates (slides from right to left) but doesn't in the view controller's view coordinates.
-    if (!CGRectEqualToRect(beginFrame, endFrame))
+    // Second condition: check if the height of the keyboard changed.
+    if (!CGRectEqualToRect(beginFrame, endFrame) || fabs(preKeyboardHeight - self.keyboardHC.constant) >= FLT_EPSILON)
```

Comment:
@SLAMAKA You can do this setting `inputView` to any kind of `UIResponder`. 

Reference: https://developer.apple.com/library/content/documentation/StringsTextFonts/Conceptual/TextAndWebiPhoneOS/InputViews/InputViews.html

Since this is not relevant to this pull request, could we move the discussion to a private email thread, feel free to drop me a message at m@cyhsu.me

Thanks.